openapi/data_repository_service.swagger.yaml

```diff
@@ -282,20 +319,52 @@ definitions:
     properties:
       object:
         $ref: '#/definitions/Object'
-  URL:
+  GetAccessURLResponse:
     type: object
-    required: ['url']
+    required: ['object']
     properties:
-      url:
+      access_url:
```

Comment:
So it looks like we can just returned signed URLs?  In the previous response that came back we returned a URL along with system_metadata, user_metadata, and authorization_metadata.  In the NIH Data Commons we needed the ability to provide more context about the URL to allow a user to make a request.  Also, in TOPMed and HCA we need to pass back native URLs to s3 and google storage.  Which will work here.  But there's no way that I can see in the current schema to pass back temporary credentials to access those native URLs.  I think this would come up with other protocols like globus or aspera, the users is going to need more than a URL.  What do you think is the best way to support this?  It feels like this is the part of the response that would need to include that info.

Reply:
Answered in [#239](https://github.com/ga4gh/data-repository-service-schemas/issues/239#issuecomment-472534495).